tests/testMap.cpp

```diff
@@ -19,6 +16,30 @@ BOOST_AUTO_TEST_CASE(myTestCase){
     token.push_back("working");
     token.push_back("okay");
 
-    BOOST_TEST(token == myMap.tokenize("This is a test -- is the code wor-king okay?"));
+    BOOST_TEST(token == Map::tokenize("This is a test -- is the code wor-king okay?"));
+    BOOST_TEST(true);
+}
+
+BOOST_AUTO_TEST_CASE(TestMap_map){
```

Comment:
Before we merge, I have some questions about this test case... 
That "testMap.txt" input file doesn't actually exist in the test folder, but the test case throws no error, and so that gets me a little concerned about how the class operates.